### PR TITLE
Allow pydeck user to modify tooltip text

### PR DESCRIPTION
**Pull request by ajduberstein**
For #3640

#### Change List
-

### CODE REVIEW

**Comment by ajduberstein** [comment]
tooltip

File: bindings/python/pydeck/pydeck/bindings/deck.py [hunk 1]
```diff
@@ -40,9 +40,13 @@ def __init__(
         mapbox_key : str, default None
             Read on initialization from the MAPBOX_API_KEY environment variable. Defaults to None if not set.
             See https://docs.mapbox.com/help/how-mapbox-works/access-tokens/#mapbox-account-dashboard
-        tooltip : bool, default True
+        tooltip : :obj:`bool` or :obj:`dict`, default True
             Boolean indicating whether or not a tooltip should be generated when hovering over a data layer
-            Individual layers must have `pickable` set to `True` to be displayed in the tooltip.
+            Alternatively, you can pass a dictionary with the optional keywords `html`, `text`, and `style`.
+            `html` is a string that sets the innerHTML value of the tooltip,
```
Comment (Pessimistress) [inline bindings/python/pydeck/pydeck/bindings/deck.py]: Explain your templating syntax.

Comment (ajduberstein) [reply bindings/python/pydeck/pydeck/bindings/deck.py]: Addressed in the sphinx PR: https://github.com/uber/deck.gl/blob/fd235cd1b625ec676bda63552617499afe7ce156/bindings/python/pydeck/docs/tooltip.rst

File: modules/jupyter-widget/src/widget-tooltip.js [hunk 2]
```diff
@@ -99,3 +105,56 @@ function toText(jsonValue) {
...
+          return null;
+        }
+        return {
+          text: substituteIn(tooltip.text, pickedInfo.object),
+          style: tooltip.style
+        };
+      };
+    }
+
+    if (tooltip.html) {
```
Comment (Pessimistress) [inline modules/jupyter-widget/src/widget-tooltip.js]: You can merge these two ifs:

```js
       return {
          html: tooltip.html && substituteIn(tooltip.html, pickedInfo.object),
          text: tooltip.text && substituteIn(tooltip.text, pickedInfo.object),
          style: tooltip.style
        };
```

**Pull request merged**